### PR TITLE
small update to readme

### PR DESCRIPTION
**Pull request by beav**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,24 @@
...
+
+# how to run with local drift-backend
+
+make a js file somewhere that looks like this
+
+```
+module.exports.routes = {
+           '/r/insights/platform/drift' : { host: 'http://localhost:8080' },
+}
+```
```
Comment (ehelms) [inline README.md]: Let's put this into the repository since it should be consistent for all users of drift-frontend to use at some point?

Comment (ehelms) [reply README.md]: Thoughts here @beav  ?

Comment (beav) [reply README.md]: I put in a PR to store it in drift-backend at https://github.com/RedHatInsights/drift-backend/pull/20/files. Once that's merged, I'll update this PR to reference that file.

**Comment by ehelms** [comment]
Can be updated now.

**Pull request merged**